src/common/CMakeLists.txt

```diff
@@ -80,15 +77,27 @@ add_subdirectory(proxybase)
 add_subdirectory(state)
 add_subdirectory(utility)
 
-target_include_directories(visitcommon PUBLIC ${VISIT_BINARY_DIR}/include)
+visit_append_list(
+    NAME  visitcommon_INCLUDES
+    ITEMS $<BUILD_INTERFACE:${VISIT_BINARY_DIR}/include>
+          $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/icons>
+          $<INSTALL_INTERFACE:${VISIT_INSTALLED_VERSION_INCLUDE}/visit/include>)
 
-# Specify the libraries to link against on Windows
-target_link_libraries(visitcommon PUBLIC ${ZLIB_LIBRARY})
 if(WIN32)
-    target_link_libraries(visitcommon PUBLIC ws2_32 shlwapi userenv )
+    visit_append_list(NAME visitcommon_DEPENDS ITEMS ws2_32 shlwapi)
```

Comment:
Just wanna confirm...no need for `userenv` anymore?

Reply:
It is actually added from common/misc/CMakeLists.txt, where the dependency comes to play. No need for it to be added twice.